insights/parsers/lvm.py

```diff
@@ -443,7 +443,7 @@ def parse_content(self, content):
 @parser(Specs.lvs_noheadings)
 class Lvs(Lvm):
     """
-    Parse the output of the `/sbin/lvs --nameprefixes --noheadings --separator='|' -a -o lv_all` command.
+    Parse the output of the `/sbin/lvs --nameprefixes --noheadings --separator='|' -a -o lv_name,lv_size,lv_attr,mirror_log,vg_name,devices,region_size,data_percent,metadata_percent,segtype,seg_monitor --config="global{locking_type=0}"` command.
```

Comment:
This is an appropriate place to include the entire command.

Reply:
sue, It is an appropriate place to add it